packages/react-magma-dom/src/components/SelectionControls/InputStyles.tsx

```diff
@@ -22,10 +21,10 @@ export const DisplayInputStyles = css`
     // active state
```

Comment:
Same here - whenever possible, use the px values from the theme instead of hardcoded ones.